tutor/src/theme.js

```diff
@@ -63,6 +63,16 @@ export const colors = {
     divider: '#cdcdcd',
   },
 
+  templates: {
+    reading: {
+      background: '#fcf6d8',
+      border: '#f4d018',
+    },
+    homework: {
+      background: '#daf3f8',
+      border: '#34bdd7',
+    },
```

Comment:
Hm, lets verify that these RGB values are correct, it's weird that readings & homework are getting new colors.

Reply:
I had wondered about this, there are some differences in what we'd use for border colors (the background would have to be new definitions because of the intended use.)

`reading` old and new:
<img width="170" alt="image" src="https://user-images.githubusercontent.com/34174/70934791-937a4800-1ff3-11ea-901a-1fc92e1a8a00.png">

<img width="169" alt="image" src="https://user-images.githubusercontent.com/34174/70934988-0a174580-1ff4-11ea-8647-32deb05376df.png">

`homework` old and new:

<img width="168" alt="image" src="https://user-images.githubusercontent.com/34174/70934865-c6244080-1ff3-11ea-8bc5-1a34af75baae.png">

<img width="165" alt="image" src="https://user-images.githubusercontent.com/34174/70935048-29ae6e00-1ff4-11ea-97c0-bb9d46d08e6e.png">